libs/common/src/main/kotlin/vilkaarsvurdering/VilkaarType.kt

```diff
@@ -1,20 +1,20 @@
 package no.nav.etterlatte.libs.common.vilkaarsvurdering
 
 enum class VilkaarType(val rekkefoelge: Int) {
```

Comment:
Vi skal ikke bare ha to ulike vilkaartyper da?

Reply:
Kunne egentlig tenkt meg det - men det krever nok ganske mye omskriving

Reply:
om du har noen gode forslag til løsning her så tar jeg gjerne innspill